### PR TITLE
PendingLoans: display delivery method

### LINKED ISSUE #261: Display delivery method for loan request in profile page
Display the delivery method for each pending loan request in the profile page of a patron because right now there is no way for the patron to see what delivery method they had chosen for a request.

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes #261 

Screenshot 2020-11-06 at 09 41 06

### CODE REVIEW

**Review by kprzerwa — COMMENTED** [review]
could we have the label in the same line as the other labels? since there is a lot of free space

**Review by jrcastro2 — APPROVED** [review]
LGTM! Very very minor: Personally I don't like that much the warehouse icon, I would like to propose the university icon instead. WDYT?

**Review by FlorianCassayre — APPROVED** [review]
Looks good!
One small detail (not related to your PR): when the label stack on top of each other, there is no margin between them.
image

**Comment by FlorianCassayre** [comment]
I agree with @jrcastro2, we use `university` at other places to represent the library.

**Review by kprzerwa — APPROVED** [review]

File: src/lib/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js [hunk 1]
```diff
@@ -177,6 +179,10 @@ class LoansListEntry extends Component {
           {DateTime.fromISO(endDate).toLocaleString()}
         </Label.Detail>
       </Label>
+      <Label basic>
+        <Icon className={delivery === 'PICKUP' ? 'university' : 'dolly'} />{' '}
+        {invenioConfig.CIRCULATION.deliveryMethods[delivery]}
```
Comment (kprzerwa) [inline src/lib/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js:184]: Minor: could you wrap it in `<Label.Detail>` ? the text is a bit too bold

Comment (ntarocco) [reply src/lib/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js:184]: @KonstantinaStoikou sorry for this, it was me. But after the last demo, it looks like that important text is not very visible in the profile page and user could get confused, at least this was the feedback we got.
The idea is to highlight very very much the loan return date, and highlight less the rest.

**Pull request merged**